Yoshi.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = "Yoshi"
-  s.version          = "3.0.0"
+  s.version          = "3.0.1"
```

Comment:
Following previous Yoshi iterations, we should increment this to 4.0.0 for Swift 5

Reply:
Gotcha :)